Packs/GitHub/Integrations/GitHub/GitHub_test.py

```diff
@@ -15,9 +14,13 @@ def load_test_data(json_path):
         return json.load(f)
 
 
+branch_pull_requests_data = load_test_data('./test_data/get-branch-pull-requests-response.json')
```

Comment:
Why this isn't in `test_list_branch_pull_requests_command`?

Reply:
will change name